helm_deploy/values-preprod.yaml

```diff
@@ -9,6 +9,11 @@ generic-service:
 
   env:
     APPLICATIONINSIGHTS_CONFIGURATION_FILE: applicationinsights.dev.json
+    API_BASE_URL_HMPPS_AUTH: "https://sign-in-preprod.hmpps.service.justice.gov.uk/auth"
+    API_BASE_URL_MANAGE_USERS: "https://manage-users-api-preprod.hmpps.service.justice.gov.uk"
+    API_BASE_URL_PRISONER_SEARCH: "https://prisoner-search-preprod.prison.service.justice.gov.uk"
+    EVENTS_BASE_URL: "https://csip-api-preprod.hmpps.service.justice.gov.uk"
```

Comment:
I think we have missed the EVENTS_BASE_URL in the dev values in both alerts and here.

Also, even though we aren't publishing any events yet, I'd add `EVENTS_PUBLISH: false` to both the dev and preprod values so that when we do start publishing events, they won't actually be published until ourselves and SYScon are ready to test them.

Reply:
EVENTS_PUBLISH added, and EVENTS_BASE_URL added to the local run script too